typer/_completion_shared.py

```diff
@@ -100,13 +100,13 @@ def install_bash(*, prog_name: str, complete_var: str, shell: str) -> Path:
     # It seems bash-completion is the official completion system for bash:
     # Ref: https://www.gnu.org/software/bash/manual/html_node/A-Programmable-Completion-Example.html
     # But installing in the locations from the docs doesn't seem to have effect
-    completion_path = Path.home() / f".bash_completions/{prog_name}.sh"
+    completion_path = Path.home() / ".bash_completions" / f"{prog_name}.sh"
```

Comment:
Note that this particular fix is not required to solve the issue on my end. I just think it's safer to always use the `Pathlib` separator `/` instead of the literal `"/"`.

Reply:
It really makes no difference because `Pathlib` perfectly understands `/` in paths 😊 it's designed to be cross-platform

Reply:
Yea I think you're right - it makes no difference because `Pathlib` will deal with the literal string correctly, too. Honestly I'd still prefer the edit for esthetic reasons (part of the same path is using `/`, the other part is using `"/"`) but I don't feel strongly. Will leave it up to Tiangolo - and either way as I said this is not actually related to the bug fix.

Reply:
Hey, I also think it's more esthetic this way 😉 just pointing out it doesn't really make it any safer